### ISSUE TITLE
Dependency on `lossless-json`

### ISSUE THREAD

**Issue by BAllendorf-OneFiber** [josdejong/svelte-jsoneditor#153]
Hey there,

I've just tried v0.7.7 and I think the dependency on `lossless-json` is broken:
```
Cannot find module 'lossless-json' imported from '/app/node_modules/svelte-jsoneditor/plugins/query/javascriptQueryLanguage.js'
```

Maybe because `lossless-json` is only a dev-dependency?

**Comment by josdejong** [comment]
Ai, thanks for reporting Benedikt. Will fix it directly.

**Comment by josdejong** [comment]
Fixed now in `v0.7.8`. 

The library should not depend on the `lossless-json` library: that is only used in the test and develop environment to try out the new `parser` option that allows passing a custom JSON parser.
